[2020-06-01 21:34:34,144] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_covid.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 21:34:34,151] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_covid.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 21:34:34,151] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 21:34:34,151] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 21:34:34,152] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 21:34:34,167] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask1> on 2020-05-30T00:00:00+00:00
[2020-06-01 21:34:34,169] {standard_task_runner.py:53} INFO - Started process 24031 to run task
[2020-06-01 21:34:34,230] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_covid.multitask1 2020-05-30T00:00:00+00:00 [running]> nineleaps
[2020-06-01 21:34:34,242] {logging_mixin.py:112} INFO - {'a': 1, 'b': 2}
[2020-06-01 21:34:34,242] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 21:34:34,246] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_covid, task_id=multitask1, execution_date=20200530T000000, start_date=20200601T160434, end_date=20200601T160434
[2020-06-01 21:34:44,135] {logging_mixin.py:112} INFO - [2020-06-01 21:34:44,134] {local_task_job.py:103} INFO - Task exited with return code 0
